redis/client.py

```diff
@@ -769,6 +770,27 @@ def lastsave(self):
         """
         return self.execute_command('LASTSAVE')
 
+    def migrate(self, host, port, key, destination_db, timeout, copy=False, replace=False, auth=None, keys=[]):
```

Comment:
I think it would be nicer if the signature would be keys, accepting either a string or a list of strings representing the names. Internally, you'll replace the 'key' with the single key, or an empty string and the `KEYS` keyword followed by the list.